server.js

```diff
@@ -14,14 +14,25 @@ try {
   pg = require("pg");
 }
 
-var paypal_query = "SELECT SUM(settle_amount)::numeric FROM paypal WHERE timestamp > $1 AND timestamp < $2 " +
-                   "AND ((type IN ('Donation', 'Payment') AND status = 'Completed') " +
-                   "OR type = 'Temporary Hold');";
-var stripe_query = "SELECT SUM(settle_amount)::numeric FROM stripe WHERE refunded = '0.00' " +
-                   "AND status = 'succeeded' AND timestamp > $1 AND timestamp < $2;";
-var bycountry_query = "SELECT country_code, sum(amount)::numeric, count(*) FROM paypal " +
-                      "WHERE timestamp > $1 AND timestamp < $2 AND country_code IS NOT NULL " +
-                      "GROUP BY country_code;";
+var paypal_total_query = `SELECT SUM(settle_amount)::numeric FROM paypal WHERE timestamp > $1 AND timestamp < $2
```

Comment:
Does all Paypal data within the date range of this campaign have a `settle_amount`? Some of the older records don't but that might not be an issue here.